packages/admin/admin/src/helpers/SlotProps.ts

```diff
@@ -0,0 +1,5 @@
+import { Theme } from "@mui/material/styles";
+import { MUIStyledCommonProps } from "@mui/system";
+import React from "react";
+
+export type SlotProps<Component extends React.ElementType> = Partial<React.ComponentProps<Component>> & MUIStyledCommonProps<Theme>;
```

Comment:
I don't think there is a need for a `theming` directory, just for this file. 

Most theming-specific things will be added in the `@comet/admin-theme` package. 
The only reason this file is in the `@comet/admin` package is, that that it might be needed by other packages, that don't currently have `@comet/admin-theme` as a dependency, e.g. `@comet/admin-date-time`.